ports/espressif/common-hal/rclcpy/std_msgs/Bool.c

```diff
@@ -0,0 +1,5 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2025 Lucian Copeland
+//
+// SPDX-License-Identifier: MIT
```

Comment:
Were these going to be omitted or moved to `shared-module/`?

Reply:
I figured I would post the "default" structure first before trying any shenanigans. I'll investigate the `microcontroller/ResetReason` example you mentioned to see if it's a better solution.